components/logo/logo.scss

```diff
@@ -0,0 +1,22 @@
+.logo {
+  font-weight: 700;
+  line-height: 1.2;
+  text-decoration: none;
+  color: #2a3840;
+  padding-top: 0.40625rem;
+  padding-bottom: 0.40625rem;
+  font-size: 1.125rem;
+
+  &:hover {
+    color: var(--primary);
+  }
+
+  & > img {
```

Comment:
You can nest it implicitly without `&` here.

It's better to avoid direct child nesting (`>`) where it's possible. It might happen you want to add a wrapper around `img` so then you will have a line more changes.